pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -68,6 +68,8 @@ type ConfigurationEditFlags struct {
 	GenerateRevisionName bool
 	ForceCreate          bool
 
+	File string
```

Comment:
I always debate using `Filename` instead of `File` since there is invariably a `File` object created somewhere later. Of course, `File` is shorter as a flag name so makes sense too... Argh.

Reply:
Following the `kubectl` example we could settle on `-f, --filename` as `--force` is not using that.

Reply:
Changed the name in https://github.com/knative/client/pull/913/commits/9a3c1b92bd85e16386ee9613296481877f774cb6.